src/analysis/retail/evoker/preservation/constants.ts

```diff
@@ -1,10 +1,36 @@
-//hots
+import SPELLS from 'common/SPELLS';
+import { TALENTS_EVOKER } from 'common/TALENTS';
+
+export const SPELL_COLORS = {
+  ECHO: '#fffb0a',
+  TA_ECHO: '#6711ad',
+  REVERSION: '#ffbb78',
+  LIVING_FLAME: '#880e0b',
+  VERDANT_EMBRACE: '#96ffd8',
+  DREAM_BREATH: '#c49c94',
+  SPIRITBLOOM: '#fe85ff',
+  EMERALD_BLOSSOM: '#ffffff',
+  FLUTTERING_SEEDLING: '#50c878',
+};
+
+// heal events that can be caused by an echo heal
+export const ECHO_HEALS = [
+  SPELLS.DREAM_BREATH_ECHO,
+  SPELLS.LIVING_FLAME_HEAL,
+  SPELLS.EMERALD_BLOSSOM_ECHO,
+  TALENTS_EVOKER.SPIRITBLOOM_TALENT,
+  TALENTS_EVOKER.VERDANT_EMBRACE_TALENT,
+  SPELLS.REVERSION_ECHO,
+  SPELLS.FLUTTERING_SEEDLINGS_HEAL,
```

Comment:
Ya It's proc'd from a echo emerald blossom. Mainly used for generating healing by spell for echo

Reply:
Thinking about this some more, if we are going to attribute fluttering seedlings from echoed emerald blossoms for this analysis, we would need to also add secondary healing gains from other talents that synergize with echo, such as; 

- Additional Golden Hour healing from echoed Reversions
- Additional Lifebind healing from echoed Verdant Embraces

Since that makes this even more unwieldy, the alternative I can think of  is to move those contributions into their own talent analysis and specify the source of the healing (echo or hardcast) in each. Or for the sake of the donut chart combine the secondary gains into the original casted source (Reversion, Verdant Embrace, Emerald Blossom, etc)